packages/assets/scss/03-organisms/_inline-links.scss

```diff
@@ -0,0 +1,79 @@
+@use "00-base/configure" as *;
+
+.ma__inline-links {
+  @media print {
+    display: none;
+  }
+
+  &__container {
+    @include ma-container;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: stretch;
+  }
+
+  &__item {
+    font-size: $fonts-xsmall;
+    display: flex;
+    align-items: center;
+    min-height: 1.9rem;
+
+    &:after {
+      content: "";
+      margin: 0 10px;
+      width: 1px;
+      height: $fonts-xsmall;
+    }
+
+    &:last-of-type:after {
+      content: none;
+    }
+
+    a {
+      line-height: 1rem;
+      text-decoration: none;
+      box-shadow: inset 0 0 white, inset 0px -1px rgba($c-primary, 0.4);
+
+      &:hover {
+        box-shadow: inset 0 0 white, inset 0px -2px rgba($c-primary, 0.7);
+      }
+    }
+
+    span {
+      white-space: nowrap;
+    }
+  }
+}
+
+//theme
```

Comment:
Can you combine the theme styles with the lines above? The direction is that we are moving away and consolidating the themes. We are going to leverage variables declared at the base level to handle theming

Reply:
done